### PR TITLE
Chore/update sdk to version 2405

### PR DESCRIPTION
**Pull request by ReinerSchinkoethe**
Update sdk to 2405 release version

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Application Example Hardware Engineering Demo
+# Application Example Hardware Engineering
 
 ## Description
-This repo contains an example project to demonstrate how to IT-like hardware configuration with SIMATIX AX.
+This reposetory contains an example project to demonstrate how to IT-like hardware configuration with SIMATIX AX.
```
Comment (BeckerStS) [inline Readme.md]: repository

File: apax.yml [hunk 2]
```diff
@@ -1,9 +1,22 @@
-name: "ae-hw-engineering-demo"
+# General information
+name: "ae-hw-engineering"
 version: 0.0.0
```
Comment (BeckerStS) [inline apax.yml:3]: 1.0.0

Comment (BeckerStS) [inline apax.yml]: 1.0.0

**Comment by BeckerStS** [comment]
@ReinerSchinkoethe  pls. also adjust/add **.gitattributes** files to the repo so the .st contents will not be interpreted as "SmallTalk"-language by GitHub xD

**Comment by BeckerStS** [comment]
This repository is not conform to our github rules for application examples ... also all the [.github/workflows] are missing, which will be executed when creating a release ... pls carefully read : https://github.com/simatic-ax/template-app  .... ReadMe.md.

File: Readme.md (hunk 1 above)
Comment (ReinerSchinkoethe) [reply Readme.md]: commit b4d8d21891bdd4bd8ce01c985bb4c492fa4dffe9

File: apax.yml (hunk 2 above)
Comment (ReinerSchinkoethe) [reply apax.yml]: commit 7c798dfa28484781c4705abf5c0b5cfa1e47b5d8

**Comment by ReinerSchinkoethe** [comment]
> @ReinerSchinkoethe pls. also adjust/add **.gitattributes** files to the repo so the .st contents will not be interpreted as "SmallTalk"-language by GitHub xD


done - commit 28cac7d4e98916ffefb86c2630853e5892147bc2

File: repolint.json [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        "type": "file-contents",
+        "options": {
+          "globsAll": [ "README*" ],
+          "content": "license",
+          "flags": "i"
+        }
+      }
+    },
+    "binaries-not-present": {
+      "level": "error",
```
Comment (BeckerStS) [inline repolint.json]:
```suggestion
      "level": "warning",
```

File: repolint.json [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      "level": "off",
+      "rule": {
+        "type": "file-contents",
+        "options": {
+          "globsAll": [ "apax.yml" ],
+          "content": "name:.*verion:.*type:.*description:.*targets:.*installStrategy:.*apaxVersion:.*Dependencies:",
+          "flags": "i"
+        }
+      }
+    },
```
Comment (BeckerStS) [inline repolint.json]:
```suggestion
```

File: repolint.json [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+          ]
+        }
+      }
+    },
+    "best-practices-badge-present": {
+      "level": "off",
+      "rule": {
+        "type": "best-practices-badge-present"
+      }
+    }
```
Comment (BeckerStS) [inline repolint.json]:
```suggestion
```

**Review by BeckerStS — APPROVED** [review]

**Pull request merged**